src/pip/_internal/req/req_tracker.py

```diff
@@ -138,13 +131,6 @@ def cleanup(self):
         for req in set(self._entries):
             self.remove(req)
 
-        if self._temp_dir is None:
-            # Did not setup the directory. No action needed.
-            logger.debug("Cleaned build tracker: %r", self._root)
-            return
-
-        # Cleanup the directory.
-        self._temp_dir.cleanup()
         logger.debug("Removed build tracker: %r", self._root)
```

Comment:
If we're removing the logging, I would drop this one too.

Reply:
Well, that wasn't intentional -- fixed in 031ed90f.